src/mono/mono/mini/interp/interp.c

```diff
@@ -3195,6 +3209,11 @@ interp_entry_from_trampoline (gpointer ccontext_untyped, gpointer rmethod_untype
 		} else {
 			size = MINT_STACK_SLOT_SIZE;
 		}
+#ifdef MONO_ARCH_HAVE_SWIFTCALL
+		if (swift_error_arg_index >= 0 && swift_error_arg_index == i) {
+			newsp->data.p = swift_error_pointer;	
```

Comment:
Shouldn't we also initialize `*swift_error_pointer` with `swift_error_data` before the call ?